src/main/java/com/stripe/model/issuing/Authorization.java

```diff
@@ -671,5 +671,35 @@ public static class VerificationData extends StripeObject {
      */
     @SerializedName("expiry_check")
     String expiryCheck;
+
+    /** The outcome of the 3D Secure authentication request. */
+    @SerializedName("three_d_secure")
+    String threeDSecure;
```

Comment:
Should this be `ThreeDSecure` instead of String?

Reply:
Oops, yes. Fixed.